mode-tree.c

```diff
@@ -776,8 +784,8 @@ mode_tree_draw(struct mode_tree_data *mtd)
 		else
 			tag = "";
 		xasprintf(&text, "%-*s%s%*s%s%s", keylen, key, start,
-		    namelen[line->depth], mti->name, tag,
-		    (mti->text != NULL) ? ": " : "" );
+		    names_all_digits[line->depth] ? namelen[line->depth] : 0,
```

Comment:
I think if I changed this to `(names_all_digits[line->depth] ? 1 : -1) * namelen[line->depth]`, it would align left, but still pad to the full width. Would that be better? I never really use more than one session, so it's not something I've though about much.